src/test/java/org/jenkinsci/plugins/ssegateway/SSEPluginIntegrationTest.java

```diff
@@ -81,4 +81,24 @@ public void test_store_and_forward() throws TaskRunnerException, IOException {
 
         gulpRunner.runIntegrationSpec("sse-plugin-store-and-forward");
     }
+
+    @Test
+    public void test_retryqueue_timeout() throws TaskRunnerException, IOException {
+        // Create a job. We will trigger a build of the job inside
+        // sse-plugin-retryqueue-timeout.js (the integration test). That build execution
+        // should trigger events to the event subscribers in the test.
+        jenkins.createFreeStyleProject("sse-gateway-test-job");
+
+        // set lifetime for retry events - default 300 sec
+        // in the integratation test we wait 20000ms until restarting the proxy
+        // -> all queued event should be removed from the queue
+        org.jenkinsci.plugins.ssegateway.sse.EventDispatcher.RETRY_QUEUE_EVENT_LIFETIME = 15;
+        // set delay for retry loop - default 100ms
+        org.jenkinsci.plugins.ssegateway.sse.EventDispatcher.RETRY_QUEUE_PROCESSING_DELAY = 500;
```

Comment:
maybe restore the initial values for next tests?

Reply:
Good Hint. Done.